src/Microsoft.AspNet.Antiforgery/AntiforgeryOptions.cs

```diff
@@ -11,16 +11,17 @@ namespace Microsoft.AspNet.Antiforgery
     public class AntiforgeryOptions
     {
         private const string AntiforgeryTokenFieldName = "__RequestVerificationToken";
+        private const string AntiforgertyTokenHeaderName = "RequestVerificationToken";
```

Comment:
This was done on purpose as a tribute to John Forgerty

![image](https://cloud.githubusercontent.com/assets/1430011/11826906/160364c4-a33d-11e5-913a-41883032f5de.png)

:+1: